.github/workflows/automation.yml

```diff
@@ -1,50 +1,55 @@
-name: Run Delegation Script
+name: Cron
+
 on:
   schedule:
-    - cron: "10 10 14 * *"
+    - cron: "0 12 1,15 * *"
```

Comment:
@okwme please let me know if you prefer a different schedule for the cron.

Reply:
this looks great!